index.bs

```diff
@@ -195,53 +203,57 @@ When the <dfn method for=AnimationWorkletGlobalScope>registerAnimator(|name|,
     7. If the result of <a>IsCallable</a>(|animate|) is false, <a>throw</a> a <a>TypeError</a> and
         abort all these steps.
 
-    8. Let |inputProperties| be the result of <a>parsing a property list</a> with name
-        "inputProperties" for |animatorCtor|. If an exception is thrown, rethrow the exception and
-        abort all these steps.
+    8. Let |elements| be the result of <a>parsing an animator slot list</a> for |animatorCtor|.
+        If an exception is thrown, rethrow the exception and abort all these steps.
 
-    9. Let |outputProperties| be the result of <a>parsing a property list</a> with name
-        "outputProperties" for |animatorCtor|. If an exception is thrown, rethrow the exception and
-        abort all these steps.
 
-    10. Let |rootInputProperties| be the result of <a>parsing a property list</a> with name
-        "rootInputProperties" for |animatorCtor|. If an exception is thrown, rethrow the exception
-        and abort all these steps.
-
-    11. Let |rootOutputProperties| be the result of <a>parsing a property list</a> with name
-        "rootOutputProperties" for |animatorCtor|. If an exception is thrown, rethrow the exception
-        and abort all these steps.
-
-
-    12. Let |definition| be a new <a>animator definition</a> with:
+    9. Let |definition| be a new <a>animator definition</a> with:
 
         - <a>animator name</a> being |name|
 
         - <a>class constructor</a> being |animatorCtor|
 
         - <a>animate function</a> being |animate|
 
-        - <a>animator input property list</a> being |inputProperties|
-
-        - <a>animator input property list</a> being |outputProperties|
+        - <a>element proxy map</a> being |elements|
 
-        - <a>animator root input property list</a> being |rootInputProperties|
 
-        - <a>animator root output property list</a> being |rootOutputProperties|
-
-
-    13. Add the key-value pair (|name| - |definition|) to the <a>animator name to animator
+    10. Add the key-value pair (|name| - |definition|) to the <a>animator name to animator
         definition map</a> of the associated <a>document</a>.
 
 
 Issue: Todo: Describe how we can read scroll offsets. The previous approach was to list
 it as an input / output property of proxies but the new idea is to have a readonly ScrollTimeline.
 
-When <dfn>parsing a property list</dfn> with name <em>name</em> for {{animatorCtor}},
+When <dfn>parsing an animator slot list</dfn> for {{animatorCtor}}, the user agent <em>must</em>
+run the following steps:
+
+    1. Let |elements| be an empty <code>sequence&lt;Dictionary></code>.
+
+    1. Let |elementsIterable| be the result of <a>Get</a>(|animatorCtor|, |elements|).
+
+    2. If |elementsIterable| is not undefined, then for each |slot| in |elementsIterable| run the
```

Comment:
Nit; make undefined bold to make it clear that it's a js concept and you aren't just abusing English grammar ;)